assets/dev/scss/editor/panel/controls/_media.scss

```diff
@@ -36,6 +36,7 @@
 
 	&__preview {
 		height: 100%;
+		min-height: 100px; // Fix for Safari 13 and other browsers that don't support aspect-ratio
```

Comment:
why don't you just revert to the original padding hack?

Reply:
Done. aspect ration 21:9 like we currently have in modern browsers.